static/src/js/containers/DatepickerContainer/__tests__/DatepickerContainer.test.js

```diff
@@ -280,4 +291,36 @@ describe('DatepickerContainer component', () => {
       expect(button).toHaveClass('rdtDisabled')
     })
   })
+
+  describe('when focusing the input', () => {
```

Comment:
I don't think we need these new tests. `describe('onTodayClick'`, `describe('onClearClick'` and `describe('when typing in the text field` should add assertions that `onMetricsTemporalFilter` was called as intended.

And `describe('when typing in the text field'` should have a new test where you click into the input and tab out of it and assert that `onMetricsTemporalFilter` was not called